src/main/frontend/components/onboarding.cljs

```diff
@@ -16,7 +16,7 @@
                               [:span.mr-1 (t :help/forum-community)]
                               (ui/icon "message-circle" {:style {:font-size 20}})]
          list
-         [{:title "Usage"
+         [{:title (t :help/title-usage)
```

Comment:
This file has non consistency. Namespace called `help`, but file called `onboarding`. According to [this](https://github.com/logseq/logseq/pull/9588#discussion_r1218030777) conversation it is confusing.
But I would leave it as is for now.